lib/builder.js

```diff
@@ -122,7 +123,7 @@ function writeOutputFile(opts, output, basePath) {
   });
 }
 
-exports.inlineSourceMap = function(output) {
+var inlineSourceMaps = exports.inlineSourceMap = function(output) {
```

Comment:
(Nit: we actually do emit only a single flattened source map, even if we generate intermediate ones. But that'll change with #51)